### PR TITLE
Fixes #21891: A namevirtualhost is mandatory when splitting vhost

### PR DESCRIPTION
**Pull request by peckpeck**
https://issues.rudder.io/issues/21891

### CODE REVIEW

**Review by fanf — APPROVED** [review]

File: relay/sources/apache/rudder-vhost.conf [hunk 1]
```diff
@@ -46,13 +46,17 @@
 # # If you change the port for one of the virtual hosts you will likely need to add
 # # Listen 8443
 #
+# # If you already have this you can remove it
+# NameVirtualHost *:443
```
Comment (amousset) [inline relay/sources/apache/rudder-vhost.conf:50]: this has no effect on apache 2.4

Comment (amousset) [reply relay/sources/apache/rudder-vhost.conf:50]: And I think we don't have 2.2 anywhere

Comment (amousset) [reply relay/sources/apache/rudder-vhost.conf:50]:
```
In 2.3.11 and later, any time an IP address and port combination is used in multiple virtual hosts, name-based virtual hosting is automatically enabled for that address.

This directive currently has no effect.
```

https://httpd.apache.org/docs/2.4/mod/core.html#namevirtualhost

Comment (amousset) [reply relay/sources/apache/rudder-vhost.conf:50]: and we already successfully split the virtual host without it, so I don't think it changes something.

**Comment by peckpeck** [comment]
Let's close it and reopen if we encouter it again

**Pull request closed without merging**